### PR TITLE
Add mingw ci and building document

### PR DESCRIPTION
**Pull request by TianZerL**
Finish #5535
https://zhuanlan.zhihu.com/p/706050397

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=5547) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=5547) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/ncnn?pullRequest=5547) it.</sub>

File: docs/how-to-build/how-to-build.md [hunk 1]
```diff
@@ -256,6 +257,26 @@ cmake --build . --config Release --target install
 ```
 
 ***
+
+### Build for Windows x64 using MinGW-w64
+
+Download MinGW-w64 toolchain from [winlibs](https://winlibs.com/) or [w64devkit](https://github.com/skeeto/w64devkit), add `bin` folder to environment variables.
+
+(optional) Download and install Vulkan SDK from <https://vulkan.lunarg.com/sdk/home>
```
Comment (nihui) [inline docs/how-to-build/how-to-build.md:265]: vulkan sdk is not needed now :)

**Comment by nihui** [comment]
add mingw build doc link to ncnn readme

图片

**Comment by nihui** [comment]
Thanks for your contribution !

**Pull request merged**